lane.py

```diff
@@ -1356,27 +1353,12 @@ def works(self, _db, facets=None, pagination=None, include_quality_tier=False):
                 mw.collection_id.in_(self.collection_ids)
             )
         qu = self.apply_filters(_db, qu, facets, pagination)
-        if qu:
```

Comment:
I'm pretty sure this conditional was only necessary because of a bad mock implementation of `apply_filters` in a test. If this method returns None instead of a real query, the calling code isn't going to work.